core/modules/drr.cc

```diff
@@ -0,0 +1,392 @@
+#include "drr.h"
+
+#include <rte_ether.h>
+#include <rte_ip.h>
+#include <rte_udp.h>
+#include <time.h>
+#include <glog/logging.h>
+
+#include <string>
+#include <cmath>
+#include <iostream>
+#include <fstream>
+
+
+#include "../mem_alloc.h"
+
+int RoundToPowerTwo(uint32_t v) {
```

Comment:
I use `RoundPowerTwo` for the llring which expects power of 2 in `uint32_t` for size. so, to keep everything in scope, I have been using `uint32_t` for `max_flow_queue_size` and `max_number_flows_`.